internal/commands/run.go

```diff
@@ -115,12 +116,15 @@ func run(cmd *cobra.Command, args []string) error {
 
 	// Enable logging
 	shouldUseSimpleRenderer := verbose || os.Getenv("CI") == "true"
-	var renderer echelon.LogRendered = renderers.NewSimpleRenderer(cmd.OutOrStdout(), nil)
+	var verboseRenderer = renderers.NewSimpleRenderer(cmd.OutOrStdout(), nil)
+	var renderer echelon.LogRendered = verboseRenderer
 	if !shouldUseSimpleRenderer {
-		interactiveRenderer := renderers.NewInteractiveRenderer(cmd.OutOrStdout(), nil)
+		interactiveRenderer := renderers.NewInteractiveRenderer(os.Stdout, nil)
```

Comment:
Why change that?

Reply:
It wasn't compiling. See this PR for details https://github.com/cirruslabs/echelon/pull/5 Echelon's interactive compiler need `stdout` now for figuring out terminal's dimentions.